index.html

```diff
@@ -15,14 +15,24 @@
         <div class="navbar">
           <a href="#" class="logo">Patrick Maina</a>
           <nav class="side-nav">
+            
             <ul class="nav-items">
```

Comment:
- [ ] After opening the mobile menu and closing it I am not able to open it again until I refresh the page. Please make sure to fix this issue and users should be able to open it every time without refreshing the page.